lib/lifecycle.fnl

```diff
@@ -0,0 +1,50 @@
+(local {:do-action do-action} (require :lib.bind))
```

Comment:
Would be nice to have a docstring with either simple explanation, or/and link to extended explanation in the docs